client/src/components/Button.js

```diff
@@ -2,6 +2,7 @@ import styled from "@emotion/styled";
 
 const Button = styled.button`
   font-family: "SFUIbold";
+  display: flex;
```

Comment:
This line has no effect because in L:12 it's set to `inline-block`.

Reply:
Thank you!

Reply:
Fix #47